spec/models/collection_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Collection, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Generally I either delete the file or delete the pending line. I use pending specs to remind myself I have work to do on the current feature. Leaving a pending spec implies that things are incomplete.
As you will almost certainly be writing specs for a collection in the near future, I'd just delete the pending line.